staging/src/k8s.io/apiserver/pkg/storage/cacher_whitebox_test.go

```diff
@@ -73,79 +77,88 @@ func TestCacheWatcherHandlesFiltering(t *testing.T) {
 		{
 			events: []*watchCacheEvent{
```

Comment:
this diff is best viewed without whitespace:
https://github.com/kubernetes/kubernetes/pull/58547/files?w=1

this makes the test data accurate (events have resource versions), and actually tests for the result we want (delete event is delivered with the resource version of the event that caused it, *not* the last modified resourceVersion of the object)